dashboard/app/models/foorm/simple_survey_form.rb

```diff
@@ -22,4 +22,13 @@ class Foorm::SimpleSurveyForm < ApplicationRecord
     'survey_data',
     'allow_multiple_submissions'
   ]
+
+  def self.find_most_recent_form_for_path(path)
+    where(path: path).last
+  end
+
+  def self.form_path_disabled?(path)
+    disabled_forms = DCDO.get('foorm_simple_survey_disabled', [])
```

Comment:
I created a new DCDO flag (`foorm_simple_survey_disabled`) and set it to the contents of the existing `foorm_misc_survey_disabled` flag.